src/Components/Postings/index.js

```diff
@@ -11,6 +12,9 @@ return (
     <div className="postings-title-wrapper">
       <h1 className="postings-title">Open Volunteer Positions</h1>
     </div>
+    <div>
+      <Navigation postings={ postings }/>
```

Comment:
I think it's good to have a search inside the navigation so we can reuse it later if we have more pages with more data